modules/signals/src/deep-signal.ts

```diff
@@ -8,7 +8,7 @@ import { IsKnownRecord } from './ts-helpers';
 
 // An extended Signal type that enables the correct typing
 // of nested signals with the `name` or `length` key.
-interface Signal<T> extends NgSignal<T> {
+export interface Signal<T> extends NgSignal<T> {
```

Comment:
This export fixes the same compilation error in TS libraries with `declaration: true` when `signalState` is used. The problem was caused by another unique symbol - Angular's `SIGNAL`.

Reply:
👍